### ISSUE TITLE
letsencrypt/internal-functions: line 84: syntax error near unexpected token `fi'

### ISSUE THREAD

**Issue by avinashbot** [dokku/dokku-letsencrypt#318]
> If you're using Dokku - especially for commercial purposes - consider donating to project development via [OpenCollective](https://opencollective.com/dokku) or [Patreon](https://www.patreon.com/dokku). Funds go to general development, support, and infrastructure costs.
>
> If you'd like to sponsor specific functionality, see the project's [Sponsoring](https://github.com/dokku/.github/blob/master/SPONSORING.md) document.
>
> If you need support for a version of Dokku that is more than a year old, your issue may be closed without an answer. Please upgrade to a recent version before filing an issue.

## Description of problem

When running most dokku letsencrypt commands, it's failing with:

```
/var/lib/dokku/plugins/available/letsencrypt/internal-functions: line 84: syntax error near unexpected token `fi'
```

Looks like you're missing a `then` from [this commit](https://github.com/dokku/dokku-letsencrypt/blob/master/internal-functions#L82-L84) from a couple hours ago:

```
  if [[ -z "$acme_port" ]];
    acme_port="$(get_available_port)"
  fi
```

Pinning the plugin to a version just before that commit works:
```
sudo dokku plugin:update letsencrypt bebcc372d3da40e9d03caf8eeb1b8a60ab1945e7
```

### How reproducible

Consistently, every time.

### Steps to Reproduce

1. Update to latest: `sudo dokku plugin:update letsencrypt`
2. Run any letsencrypt command: `dokku letsencrypt:active myappname`
3. "syntax error near unexpected token `fi'"

#### Actual Results

Got an error.

#### Expected Results

Not an error.

### `dokku report APP_NAME` output

> This is required! Issues missing this information may be closed.
>
> For problems affecting all applications, the report output for a broken application is useful for our debugging.
> In these cases, you may run `dokku report` without any arguments to display the top-level reporting information.

<details>
<summary>Full report</summary>

-----> uname: Linux avnsh 5.4.0-152-generic #169-Ubuntu SMP Tue Jun 6 22:23:09 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
-----> memory: 
                     total        used        free      shared  buff/cache   available
       Mem:           3919         693         113         174        3112        2803
       Swap:             0           0           0
-----> docker version: 
       Client: Docker Engine - Community
        Version:           19.03.13
        API version:       1.40
        Go version:        go1.13.15
        Git commit:        4484c46d9d
        Built:             Wed Sep 16 17:02:36 2020
        OS/Arch:           linux/amd64
        Experimental:      false
       
       Server: Docker Engine - Community
        Engine:
         Version:          19.03.13
         API version:      1.40 (minimum version 1.12)
         Go version:       go1.13.15
         Git commit:       4484c46d9d
         Built:            Wed Sep 16 17:01:06 2020
         OS/Arch:          linux/amd64
         Experimental:     false
        containerd:
         Version:          1.3.7
         GitCommit:        8fba4e9a7d01810a393d5d25a3621dc101981175
        runc:
         Version:          1.0.0-rc10
         GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
        docker-init:
         Version:          0.18.0
         GitCommit:        fec3683
-----> docker daemon info: 
       Client:
        Debug Mode: true
       
       Server:
        Containers: 20
         Running: 5
         Paused: 0
         Stopped: 15
        Images: 88
        Server Version: 19.03.13
        Storage Driver: overlay2
         Backing Filesystem: extfs
         Supports d_type: true
         Native Overlay Diff: true
WARNING: No swap limit support
        Logging Driver: json-file
        Cgroup Driver: cgroupfs
        Plugins:
         Volume: local
         Network: bridge host ipvlan macvlan null overlay
         Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
        Swarm: inactive
        Runtimes: runc
        Default Runtime: runc
        Init Binary: docker-init
        containerd version: 8fba4e9a7d01810a393d5d25a3621dc101981175
        runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
        init version: fec3683
        Security Options:
         apparmor
         seccomp
          Profile: default
        Kernel Version: 5.4.0-152-generic
        Operating System: Ubuntu 20.04.6 LTS
        OSType: linux
        Architecture: x86_64
        CPUs: 2
        Total Memory: 3.828GiB
        Name: avnsh
        ID: J37P:KGNZ:ISWM:4XII:PGDW:7SWJ:KYMH:QBM6:DR2B:FN53:KSND:W6Q5
        Docker Root Dir: /var/lib/docker
        Debug Mode: false
        Registry: https://index.docker.io/v1/
        Labels:
        Experimental: false
        Insecure Registries:
         127.0.0.0/8
        Live Restore Enabled: false
       
-----> git version: git version 2.25.1
-----> sigil version: 0.9.0build+bc921b7
-----> herokuish version: 
       herokuish: v0.6.1
       buildpacks:
         heroku-buildpack-multi     v1.2.0
         heroku-buildpack-ruby      v256
         heroku-buildpack-nodejs    v218
         heroku-buildpack-clojure   v90
         heroku-buildpack-python    v234
         heroku-buildpack-java      v72
         heroku-buildpack-gradle    v39
         heroku-buildpack-scala     v96
         heroku-buildpack-play      v26
         heroku-buildpack-php       v237
         heroku-buildpack-go        v176
         heroku-buildpack-nginx     v23
         buildpack-null             v3
-----> dokku version: dokku version 0.28.4
-----> plugn version: plugn: 0.13.0build+fd5297a
-----> dokku plugins: 
         00_dokku-standard    0.28.4 enabled    dokku core standard plugin
         20_events            0.28.4 enabled    dokku core events logging plugin
         app-json             0.28.4 enabled    dokku core app-json plugin
         apps                 0.28.4 enabled    dokku core apps plugin
         builder              0.28.4 enabled    dokku core builder plugin
         builder-dockerfile   0.28.4 enabled    dokku core builder-dockerfile plugin
         builder-herokuish    0.28.4 enabled    dokku core builder-herokuish plugin
         builder-lambda       0.28.4 enabled    dokku core builder-lambda plugin
         builder-null         0.28.4 enabled    dokku core builder-null plugin
         builder-pack         0.28.4 enabled    dokku core builder-pack plugin
         buildpacks           0.28.4 enabled    dokku core buildpacks plugin
         caddy-vhosts         0.28.4 enabled    dokku core caddy-vhosts plugin
         certs                0.28.4 enabled    dokku core certificate management plugin
         checks               0.28.4 enabled    dokku core checks plugin
         common               0.28.4 enabled    dokku core common plugin
         config               0.28.4 enabled    dokku core config plugin
         cron                 0.28.4 enabled    dokku core cron plugin
         docker-options       0.28.4 enabled    dokku core docker-options plugin
         domains              0.28.4 enabled    dokku core domains plugin
         enter                0.28.4 enabled    dokku core enter plugin
         git                  0.28.4 enabled    dokku core git plugin
         letsencrypt          0.20.1 enabled    Automated installation of let's encrypt TLS certificates
         logs                 0.28.4 enabled    dokku core logs plugin
         network              0.28.4 enabled    dokku core network plugin
         nginx-vhosts         0.28.4 enabled    dokku core nginx-vhosts plugin
         plugin               0.28.4 enabled    dokku core plugin plugin
         postgres             1.11.2 enabled    dokku postgres service plugin
         proxy                0.28.4 enabled    dokku core proxy plugin
         ps                   0.28.4 enabled    dokku core ps plugin
         redis                1.22.0 enabled    dokku redis service plugin
         registry             0.28.4 enabled    dokku core registry plugin
         repo                 0.28.4 enabled    dokku core repo plugin
         resource             0.28.4 enabled    dokku core resource plugin
         run                  0.28.4 enabled    dokku core run plugin
         scheduler            0.28.4 enabled    dokku core scheduler plugin
         scheduler-docker-local 0.28.4 enabled    dokku core scheduler-docker-local plugin
         scheduler-null       0.28.4 enabled    dokku core scheduler-null plugin
         shell                0.28.4 enabled    dokku core shell plugin
         ssh-keys             0.28.4 enabled    dokku core ssh-keys plugin
         storage              0.28.4 enabled    dokku core storage plugin
         trace                0.28.4 enabled    dokku core trace plugin
         traefik-vhosts       0.28.4 enabled    dokku core traefik-vhosts plugin
=====> appname app-json information
       App json computed selected:    app.json
       App json global selected:      app.json
       App json selected:             
=====> appname app information
       App created at:                1693159558
       App deploy source:             appname
       App deploy source metadata:    appname
       App dir:                       /home/dokku/appname
       App locked:                    false
=====> appname builder information
       Builder build dir:             
       Builder computed build dir:    
       Builder computed selected:     
       Builder global build dir:      
       Builder global selected:       
       Builder selected:              
=====> appname builder-dockerfile information
       Builder dockerfile computed dockerfile path: Dockerfile               
       Builder dockerfile global dockerfile path: Dockerfile               
       Builder dockerfile dockerfile path:                          
=====> appname builder-lambda information
       Builder lambda computed lambdayml path: lambda.yml               
       Builder lambda global lambdayml path: lambda.yml               
       Builder lambda lambdayml path:                          
=====> appname builder-pack information
       Builder pack computed projecttoml path: project.toml             
       Builder pack global projecttoml path: project.toml             
       Builder pack projecttoml path:                          
=====> appname buildpacks information
       Buildpacks computed stack:     gliderlabs/herokuish:latest-20
       Buildpacks global stack:       
       Buildpacks list:               
       Buildpacks stack:              
=====> appname ssl information
       Ssl dir:                       /home/dokku/appname/tls
       Ssl enabled:                   true                     
       Ssl hostnames:                 avn.sh          
       Ssl expires at:                Nov 25 17:05:46 2023 GMT 
       Ssl issuer:                    C = US, O = Let's Encrypt, CN = R3
       Ssl starts at:                 Aug 27 17:05:47 2023 GMT 
       Ssl subject:                   subject=CN = avn.sh
       Ssl verified:                  self signed              
=====> appname checks information
       Checks disabled list:          none                     
       Checks skipped list:           none                     
=====> appname cron information
       Cron task count:               0
=====> appname docker options information
       Docker options build:          --link dokku.postgres.appname-14:dokku-postgres-appname-14 --link dokku.redis.appname:dokku-redis-appname --link dokku.redis.appname:dokku-redis-appname 
       Docker options deploy:         --link dokku.postgres.appname-14:dokku-postgres-appname-14 --link dokku.redis.appname:dokku-redis-appname --link dokku.redis.appname:dokku-redis-appname --restart=on-failure:10 
       Docker options run:            --link dokku.postgres.appname-14:dokku-postgres-appname-14 --link dokku.redis.appname:dokku-redis-appname --link dokku.redis.appname:dokku-redis-appname 
=====> appname domains information
       Domains app enabled:           true                     
       Domains app vhosts:            avn.sh          
       Domains global enabled:        true                     
       Domains global vhosts:         avnsh-v2                  
=====> appname git information
       Git deploy branch:             main                     
       Git global deploy branch:      master                   
       Git keep git dir:              false                    
       Git rev env var:               GIT_REV                  
       Git sha:                       d0f9450                  
       Git last updated at:           1691146208               
=====> appname letsencrypt information
       Letsencrypt active:            true                     
       Letsencrypt autorenew:         false                    
       Letsencrypt computed dns provider:                          
       Letsencrypt global dns provider:                          
       Letsencrypt dns provider:                               
       Letsencrypt computed email:    smlt@avn.sh              
       Letsencrypt global email:      smlt@avn.sh              
       Letsencrypt email:                                      
       Letsencrypt expiration:        1700931946               
       Letsencrypt computed graceperiod: 2592000                  
       Letsencrypt global graceperiod:                          
       Letsencrypt graceperiod:                                
       Letsencrypt computed lego docker args:                          
       Letsencrypt global lego docker args:                          
       Letsencrypt lego docker args:                           
       Letsencrypt computed server:   https://acme-v02.api.letsencrypt.org/directory
       Letsencrypt global server:                              
       Letsencrypt server:                                     
=====> appname logs information
       Logs computed max size:        2g
       Logs global max size:          10m
       Logs global vector sink:       
       Logs max size:                 2g
       Logs vector sink:              
=====> appname network information
       Network attach post create:           
       Network attach post deploy:           
       Network bind all interfaces:          false
       Network computed attach post create:  
       Network computed attach post deploy:  
       Network computed bind all interfaces: false
       Network computed initial network:     
       Network computed tld:                 
       Network global attach post create:    
       Network global attach post deploy:    
       Network global bind all interfaces:   false
       Network global initial network:       
       Network global tld:                   
       Network initial network:              
       Network static web listener:          
       Network tld:                          
       Network web listeners:                172.17.0.6:4000
=====> appname nginx information
       Nginx access log format:                                
       Nginx access log path:         /var/log/nginx/appname-access.log
       Nginx bind address ipv4:                                
       Nginx bind address ipv6:       ::                       
       Nginx client max body size:                             
       Nginx disable custom config:   false                    
       Nginx error log path:          /var/log/nginx/appname-error.log
       Nginx global hsts:             true                     
       Nginx computed hsts:           true                     
       Nginx hsts:                                             
       Nginx hsts include subdomains: true                     
       Nginx hsts max age:            15724800                 
       Nginx hsts preload:            false                    
       Nginx proxy buffer size:       4096                     
       Nginx proxy buffering:         on                       
       Nginx proxy buffers:           8 4096                   
       Nginx proxy busy buffers size: 8192                     
       Nginx proxy read timeout:      60s                      
       Nginx last visited at:         1693159876               
       Nginx x forwarded for value:   $remote_addr             
       Nginx x forwarded port value:  $server_port             
       Nginx x forwarded proto value: $scheme                  
       Nginx x forwarded ssl:                                  
=====> appname proxy information
       Proxy enabled:                 true
       Proxy port map:                http:80:4000 https:443:4000
       Proxy type:                    nginx
=====> appname ps information
       Deployed:                      true
       Processes:                     1
       Ps can scale:                  true
       Ps computed procfile path:     Procfile
       Ps global procfile path:       Procfile
       Ps procfile path:              
       Ps restart policy:             on-failure:10
       Restore:                       true
       Running:                       true
       Status web 1:                  running (CID: f57539be163)
=====> appname registry information
       Registry computed image repo:      dokku/appname
       Registry computed push on release: false
       Registry computed server:          
       Registry global push on release:   
       Registry global server:            
       Registry image repo:               
       Registry push on release:          
       Registry server:                   
       Registry tag version:              
=====> appname resource information
=====> appname scheduler information
       Scheduler computed selected:   docker-local
       Scheduler global selected:     docker-local
       Scheduler selected:            
=====> appname scheduler-docker-local information
       Scheduler docker local disable chown:                          
       Scheduler docker local init process: true                     
       Scheduler docker local parallel schedule count:                          
=====> appname storage information
       Storage build mounts:                                   
       Storage deploy mounts:                                  
       Storage run mounts:  

</details>

### How (deb/make) and where (AWS, VirtualBox, physical, etc.) was Dokku installed?:

It's been a while, but I'm mostly sure it's DigitalOcean's starter image 

### Additional information

- App container inspect output (if applicable) via `dokku ps:inspect APP_NAME`
- The nginx configuration (if applicable) via `dokku nginx:show-config APP_NAME`
- Link to the exact repository being deployed (if possible/applicable):
- Output of failing Dokku commands after running `dokku trace:on`
  (BEWARE: `trace:on` will print environment variables for some commands, be sure you're not exposing any sensitive information when posting issues. You may replace these values with XXXXXX):

**Comment by bmatusiak** [comment]
same

**Comment by asdaviid** [comment]
experiencing same. how can i install an older version?

**Comment by iabdulin** [comment]
same

**Comment by josegonzalez** [comment]
Well at least people are upgrading!

**Comment by josegonzalez** [comment]
Released in 0.20.3.

**Comment by bmatusiak** [comment]
I think that was the quickest bug fix in history on GitHub I've ever seen

Thanks!
